src/StatusQ/Controls/StatusBaseInput.qml

```diff
@@ -54,91 +56,107 @@ Item {
+                    color: Theme.palette.directColor1
 
-                onCursorRectangleChanged: flick.ensureVisible(cursorRectangle)
-                wrapMode: TextEdit.NoWrap
+                    onCursorRectangleChanged: flick.ensureVisible(cursorRectangle)
+                    wrapMode: TextEdit.NoWrap
 
-                Keys.onReturnPressed: {
-                    if (multiline) {
-                        event.accepted = false
-                    } else {
-                        event.accepted = true
+                    Keys.onReturnPressed: {
+                        if (multiline) {
+                            event.accepted = false
+                        } else {
+                            event.accepted = true
+                        }
                     }
-                }
 
-                Keys.onEnterPressed: {
-                    if (multiline) {
-                        event.accepted = false
-                    } else {
-                        event.accepted = true
+                    Keys.onEnterPressed: {
+                        if (multiline) {
+                            event.accepted = false
+                        } else {
+                            event.accepted = true
+                        }
                     }
-                }
-
-                StatusBaseText {
-                    id: placeholder
-                    visible: edit.text.length === 0
-                    anchors.left: parent.left
-                    anchors.right: parent.right
-                    font.pixelSize: 15
 
-                    elide: StatusBaseText.ElideRight
-                    font.family: Theme.palette.baseFont.name
-                    color: Theme.palette.baseColor1
+                    StatusBaseText {
+                        id: placeholder
+                        visible: edit.text.length === 0
+                        anchors.left: parent.left
+                        anchors.right: parent.right
+                        anchors.verticalCenter: parent.verticalCenter
+                        font.pixelSize: 15
+
+                        elide: StatusBaseText.ElideRight
+                        font.family: Theme.palette.baseFont.name
+                        color: statusBaseInput.enabled ? Theme.palette.baseColor1 :
+                            Theme.palette.directColor6
+                    }
                 }
             }
         } // Flickable
 
-        MouseArea {
-            anchors.fill: parent
-            cursorShape: Qt.IBeamCursor
-            enabled: !edit.activeFocus
-            onClicked: edit.forceActiveFocus()
-        }
     } // Rectangle
 
     StatusFlatRoundButton {
         id: clearButtton
-        visible: edit.text.length != 0 && statusBaseInput.clearable && !statusBaseInput.multiline
+        visible: edit.text.length != 0 && 
+            statusBaseInput.clearable && 
+            !statusBaseInput.multiline &&
+            edit.activeFocus
```

Comment:
you probably now better our UX guidelines, but if the field is not empty and has no active focus, then maybe I should still see the clear button? in order to quickly clear it by mouse click? otherwise I need to clicks: one to set focus, the second is to click on this button.

Reply:
Looping in @simonricoo to get his view on this. Design system shows clear button should only be rendered when active focus

Reply:
OK. Also I just tested how this works and found this button too small and too hard to point and click. I would wish it to have larger hotspot area.

Reply:
Created an issue for this as it's outside of the scope of this PR: https://github.com/status-im/StatusQ/issues/294